stripper/ze_deadcore_k3.cfg

```diff
@@ -1,3 +1,99 @@
+;remove laser slow
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "escape_laser_hurt1"
+	}
+	delete:
+	{
+		"OnStartTouch" "speedModifySpeed0.650-1"
+		"OnStartTouch" "speedModifySpeed0.650.2-1"
+		"OnStartTouch" "speedModifySpeed0.650.4-1"
+		"OnStartTouch" "speedModifySpeed0.650.6-1"
+		"OnStartTouch" "speedModifySpeed0.650.8-1"
+		"OnStartTouch" "speedModifySpeed0.651-1"
+		"OnStartTouch" "speedModifySpeed0.651.2-1"
+		"OnStartTouch" "speedModifySpeed0.651.4-1"
+		"OnStartTouch" "speedModifySpeed0.651.6-1"
+		"OnStartTouch" "speedModifySpeed0.651.8-1"
+		"OnStartTouch" "speedModifySpeed12-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "escape_laser_hurt2"
+	}
+	delete:
+	{
+		"OnStartTouch" "speedModifySpeed0.650-1"
+		"OnStartTouch" "speedModifySpeed0.650.2-1"
+		"OnStartTouch" "speedModifySpeed0.650.4-1"
+		"OnStartTouch" "speedModifySpeed0.650.6-1"
+		"OnStartTouch" "speedModifySpeed0.650.8-1"
+		"OnStartTouch" "speedModifySpeed0.651-1"
+		"OnStartTouch" "speedModifySpeed0.651.2-1"
+		"OnStartTouch" "speedModifySpeed0.651.4-1"
+		"OnStartTouch" "speedModifySpeed0.651.6-1"
+		"OnStartTouch" "speedModifySpeed0.651.8-1"
+		"OnStartTouch" "speedModifySpeed12-1"
+	}
+}
+
+;fix rare case where red NPC targeting breaks item filternames
+add:
+{
+	"classname" "filter_activator_name"
+	"targetname" "ignore_cannon1"
+	"filtername" "cannon1"
+	"Negated" "1"
+}
+
+add:
+{
+	"classname" "filter_activator_name"
+	"targetname" "ignore_cannon2"
+	"filtername" "cannon2"
+	"Negated" "1"
+}
+
+add:
+{
+	"classname" "filter_activator_name"
+	"targetname" "ignore_cannon3"
+	"filtername" "cannon3"
+	"Negated" "1"
+}
+
+add:
+{
+	"classname" "filter_multi"
+	"targetname" "ignore_cannon"
+	"Filter01" "human"
+	"Filter02" "ignore_cannon1"
+	"Filter03" "ignore_cannon2"
+	"Filter04" "ignore_cannon3"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "enemy1_filtertrigger1"
+		"origin" "-11768 -14220 -13933.5"
+	}
+	replace:
+	{
+		"filtername" "ignore_cannon"
+	}
+}
```

Comment:
Wouldn't this mean cannon holders are immune from red NPC? You should just move one of these filters to a different system (context, vscript) so they will never conflict. See how this is done in [escapetheworld](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/master/stripper/ze_escapetheworld_a33.cfg#L111).

Reply:
Changed to context filters for boss targetting. Tested just fine.